applications/dashboard/models/class.assetmodel.php

```diff
@@ -286,6 +286,10 @@ public static function cssPath($filename, $folder = '', $themeType = '') {
 
         // 3. Check the theme.
         $theme = Gdn::themeManager()->themeFromType($themeType);
+
+        // Let override theme dynamically
+        if (isset($dynamicCurrentTheme) && $dynamicCurrentTheme != $theme) $theme = $dynamicCurrentTheme;
```

Comment:
Our coding standards require the use of a new line + braces on all logic statements. So always:

```
if (condition) {
   // do this
}
```

Reply:
Fine, will do.